### ISSUE TITLE
Redux Framework won't load options in front view after activate plugin, only after click on save btn in theme options

### ISSUE THREAD

**Issue by yupii1986** [reduxframework/redux-framework#3630]
Hi,
I have my theme made with Redux Framework, problem is after i enabled plugin redux, i can see my theme options in top bar, but in site in front are not loaded, to have my front view ok, i need to go to my theme options and to hit SAVE, after this all are loaded.

Is there a way that all options in front to be loaded automatically after enable plugin redux ?

**Comment by yupii1986** [comment]
Why is closed? Can i have any advise if is ok like this or how can i fix it?

**Comment by kprovance** [comment]
If you had read the text in the new issue window, you'd understand.  We close tickets that do not meet minimum guidelines for free support.

**Comment by yupii1986** [comment]
Where to read? I don't see anything about this, only closed

I don't want you to fix my theme, just to offer your advice.
Is normal to be like this or should load all from theme option after plugin redux in activate?

In my case, i need to go to theme option and click on save button to load all from theme option

thanks

**Comment by kprovance** [comment]
https://github.com/reduxframework/redux-framework/issues/new?template=dev.md

**Comment by yupii1986** [comment]
so you can't just offer an answer to my question?

**Comment by kprovance** [comment]
Not possible.  The answer is:  I don't know.  Why?  You didn't offer any of the *required* information.  Sorry.  I wish I could do better, but it's up to you to help us help you.  If you want to pay for premium support, then I'lls pend an hour asking the individual question.  But for free support, you must supply that information to us...and specific instructions were given.  Good luck.

**Comment by yupii1986** [comment]
Sorry but what are _required_ information ?
I use latest version stable of redux framework,
I made my theme using underscore and redux plugin

as i said, in admin i have my theme options panel made with redux, all good here.

Problem is only first time after i install plugin redux, and i go to home page, is not loaded. Only after i click on button save from theme options is loaded in front.

Is this normal for redux?

Or if i pay you 100usd, you will fix it to load in front after plugin acitvate?  without need to click on btn save from theme option.

**Comment by dovy** [comment]
@yupii1986 we have a lot of requests come through. Without the support URL we can't begin to provide any support. Oft times we go down the road of helping someone and we just need a support URL to tell the problem.

Hence what Kevin is trying to say. We cannot provide free support unless a support URL is generated. It saves us all time and makes everyone much happier.

You question is in regards to the default options. My guess is you're using Redux at a diff init hook, but I can't tell without the support URL.

Could you please read the notice when you create an issue and follow the instructions? Thank you!

**Comment by yupii1986** [comment]
Sorry,
Was from my old code.
Is fixed. Really  this helperme "diff init hook"

Cheers

PS: new version will be also free ?

**Comment by dovy** [comment]
The new core version will be free. We will be offering a premium version with a ton more goodies all bundled.  ;)
